website/versioned_docs/version-6.26.3/plugin-syntax-dynamic-import.md

```diff
@@ -5,6 +5,14 @@ sidebar_label: syntax-dynamic-import
 original_id: babel-plugin-syntax-dynamic-import
 ---
 
+`babel-plugin-syntax-dynamic-import` is needed to enable support for parsing `import()`
+
+**Usage notes:**
```

Comment:
> If you are using `babel-preset-env`, it's automatically handled

I don't think this applies for v6?

> Otherwise, you need `babel-plugin-proposal-dynamic-import`

`babel-plugin-proposal-dynamic-import` doesn't exist :)

Should we link to https://github.com/airbnb/babel-plugin-dynamic-import-node?

Reply:
😆  oops!

Yeah maybe something like:
> Otherwise, consider using [babel-plugin-dynamic-import-node](https://github.com/airbnb/babel-plugin-dynamic-import-node).

@nicolo-ribaudo Just just verify, can you confirm that babel-preset-env (v6) does not automatically handle this? If not, I'll strike that line.

Reply:
Yes, it's only supported in Babel 7 (7.5.0)

Reply:
Great! I'll strike that line.